omisego-sdk/src/main/java/co/omisego/omisego/model/transaction/consumption/TransactionConsumption.kt

```diff
@@ -146,3 +148,21 @@ data class TransactionConsumption(
         return other is TransactionConsumption && other.id == id
     }
 }
+
+/**
+ * An extension function that used the id from `TransactionConsumption` object to approve the transaction
+ *
+ * @param omgAPIClient the [OMGAPIClient] object in your application to used for approve the transaction
+ * @return The [OMGCall<TransactionConsumption>] object that you need to call enqueue method to actually make a approve request to the API
```

Comment:
that you need to call enqueue method to actually make a approve request to the API
-> that you need to call enqueue method on to actually perform the request to the API

Reply:
Fixed